src/tsp-client.spec.ts

```diff
@@ -65,6 +65,32 @@ for (const [serverName, server] of Object.entries({ executableServer, moduleServ
             assert.equal(references.body!.symbolName, 'doStuff');
         }).timeout(10000);
 
+        it('inlayHints', async () => {
+            const f = filePath('module2.ts');
+            server.notify(CommandTypes.Open, {
+                file: f,
+                fileContent: readContents(f)
+            });
+            await server.request(CommandTypes.Configure, {
+                preferences: {
+                    // @ts-expect-error preference exist
+                    includeInlayFunctionLikeReturnTypeHints: true
+                }
+            });
+            const inlayHints = await server.request(
```

Comment:
I'm sorry if I lead you the wrong path but those tests should actually be in `lsp-server.spec.ts` because we want to test our own API and not TS' one.